packaging/linux/rpm/template.go

```diff
@@ -435,21 +435,40 @@ func (w *specWrapper) BuildSteps() fmt.Stringer {
 	return b
 }
 
+func systemdPreUnScript(unitName string, cfg dalec.SystemdUnitConfig) string {
+	// if service isn't explicitly specified as enabled in the spec,
+	// then we don't need to do anything in the preun script
+	if !cfg.Enable {
```

Comment:
Do we need to check if it should be stopped before uninstall even if its not enabled?